models/curvefi/optimism/curvefi_optimism_pools.sql

```diff
@@ -1,7 +1,7 @@
 {{ config(
     alias = 'pools',
     materialized = 'incremental',
-    incremental_strategy = 'insert_overwrite',
+    incremental_strategy = 'merge',
```

Comment:
Hi @MSilb7 , I have changed the incremental_strategy in the last commit, as `'insert_overwrite'` is invalid for incremental strategy, we can either use `'append'` or '`merge`' with incremental view.

Reply:
got it, yeah this is a weird one